### DISCUSSION TITLE
New folder is created outside the current/selected folder.

### DISCUSSION THREAD

**Discussion by EmadFathy** [iterate-ch/cyberduck#16295] [Q&A]
When creating a new folder it creates it in the same level of the current/selected folder, instead of inside the current folder.

Update:
- Sorry to forget to mention, I mean the App on Windows platform.

**Comment by dkocher** [comment]
This is expected behaviour like in Finder.app.

**Reply by EmadFathy** [reply]
I know it runs on Mac and Windows, may be for the Mac users this is familiar, but for Windows user this is very disturbing.

**Comment by khart20** [comment]
Along with this problem, I also don't seem to be able to upload files into the selected folder.  It always uploads them in the same directory as the selected folder and then I have to move them.  Which takes a long time for larger files and it just not good workflow.  Am I missing something?  Right now I'm creating a new placeholder file and moving into the folder and then uploading the things I really want to upload.
